tests/solidity_test_helpers/ReentrancyGuardHelper.sol

```diff
@@ -0,0 +1,21 @@
+pragma solidity ^0.4.13;
+
+import 'libraries/ReentrancyGuard.sol';
+
```

Comment:
2 Lines before the contract definition. You can also test this style enforcement by using solium locally with this command: `solium --dir=source/contracts`

Reply:
what is meant by 'test this style enforcement' also I don't have an executable solium. I can see if I can download it on macos.

Reply:
Solium is a command line tool used to test style guidelines for Solidity. The easiest way to install it would be to use npm.

Reply:
command
>npx solium --dir=source/contracts
No issues found.

Is that what I'm suppose to see?

Reply:
```
npm install
npx solium --dir=source/contracts
```
(assumes Node 7+ I believe)

Reply:
Yes, that means everything is good. If you have only a single blank line between `import` and `contracts`  and you still see "No issues found" then something is wrong though.

Reply:
I'm realizing now our CI and the default command we use `dir=source/contracts` won't analyze the contracts in `tests`. That's why CI is passing and this doesn't come up.

Reply:
Ah.  If someone wants to update the docs and Dockerfile and maybe add an `npm lint` command I'll approve the PR, but I don't particularly care that much so I probably won't do this myself.  😄